apps/earn-protocol/components/layout/Navigation/NavigationWrapper.tsx

```diff
@@ -56,6 +56,10 @@ export const NavigationWrapper: FC<NavigationWrapperProps> = ({ panels }) => {
     return {}
   }, [wallet])
 
+  if (currentPath == null) {
```

Comment:
I don't get this part, could you explain?

Reply:
I got this issue after merge with dev, just fix for null type